examples/swig_purepython_boids_spatial3D/boids_spatial3D.py

```diff
@@ -0,0 +1,404 @@
+
+from textwrap import indent
+from pyflamegpu import *
+import pyflamegpu.codegen
+import sys, random, math
+
+
```

Comment:
Would benefit from a file-level docblock describing the file 

i.e. something like:

```suggestion
"""
  FLAME GPU 2 implementation of boids in python 3, using python to define agent functions rather than CUDA C++ RTC.
  This is based on the FLAME GPU 1 implementation, but with dynamic generation of agents.
  Agents are also clamped to be within the environment bounds, rather than wrapped as in FLAME GPU 1.
"""
```